src/server/getters/get-reporting-fees.ts

```diff
@@ -62,6 +101,127 @@ interface RepStakeResults {
   lostRep: BigNumber;
 }
 
+function getUniverseAndParentUniverse(db: Knex, reporter: Address, universe: Address|null, feeWindow: Address|null, callback: (err: Error|null, result?: UniverseAndParentUniverse) => void) {
+  const query = db("fee_windows")
+  .select("universes.universe", "universes.parentUniverse")
+  .join("universes", "fee_windows.universe", "universes.universe")
+  .groupBy("fee_windows.universe");
+  if (universe != null) {
+    query.where("fee_windows.universe", universe);
+  } else if (feeWindow != null) {
+    query.where("fee_windows.feeWindow", feeWindow);
+  }
+  query.asCallback((err, universeAndParentUniverse: Array<UniverseAndParentUniverse>) => {
```

Comment:
instead of `select()` use `first()`. It does a `LIMIT 1` and makes your callback value a single object (or null) instead of an array